docs/internals/optimizer.rst

```diff
@@ -295,11 +295,11 @@ backtracking.
 All components of the Yul-based optimizer module are explained below.
 The following transformation steps are the main components:
 
-- SSA Transform
-- Common Subexpression Eliminator
-- Expression Simplifier
-- Unused Assign Eliminator
-- Full Inliner
+- SSATransform
+- CommonSubexpressionEliminator
+- ExpressionSimplifier
+- UnusedAssignEliminator
+- FullInliner
```

Comment:
This list actually seems out of date. Or is e.g. UnusedStoreRemover or LoadResolver not a major component?

Reply:
Well, arguable - the distinction between "main" components and otherwise is blurry anyways and the UnusedStoreRemover and LoadResolver are rather weak in the sense that they only kick in under rather specific circumstances so far. Meaning: I'd not consider too relevant what we list here.